Sources/App/Models/Karma/KarmaStatus.swift

```diff
@@ -6,17 +6,17 @@
 //
 
 import Foundation
-import FluentSQLite
+import FluentPostgreSQL
 import Vapor
 
 /// Associate a karma (rating) with a unique entity
-final class KarmaStatus: SQLiteStringModel {
+final class KarmaStatus: PostgreSQLStringModel {
     var id: String?
     
     var count: Int
-    var type: StatusType
+    var type: String
```

Comment:
If Postgres doesn't allow using the enum directly, I would make this private and have a public computed property that's like 
```
var type: KarmaStatusType = { 
    get {
        return KarmaStatusType(rawValue: _type) ?? .other }
    }
    set {
        _type = newValue.rawValue
    }
}
```

Reply:
It does but it makes the column type a `jsonb` instead of `text` and it was causing problems. I will change it to be a computed though